ports/servoshell/desktop/app_state.rs

```diff
@@ -526,13 +526,6 @@ impl WebViewDelegate for RunningAppState {
         self.inner_mut().need_present = true;
     }
 
-    fn notify_event_delivered(&self, webview: servo::WebView, event: CompositorEventVariant) {
-        if let CompositorEventVariant::MouseButtonEvent = event {
-            webview.raise_to_top(true);
-            webview.focus();
-        }
-    }
-
```

Comment:
This was a feature used for the MDI WebView experiment. I've removed it because we are gradually moving to a model where the embedder manages focus and draw order itself and each `WebView` is a separate "widget."

Reply:
I just noticed this is removed. How do we focus webview after this PR?

Reply:
Just saw #35454. I suppose real fix is to make webview can be focused instead of just elements.

Reply:
> I just noticed this is removed. How do we focus webview after this PR?

Focus should be handled by the embedder. So in minibrowser, focus switches when switching tabs. This code was just for the MDI experiment that isn't used any longer.